src/encord_active/cli/print.py

```diff
@@ -80,8 +82,9 @@ def print_ontology(
     json_ontology = json.dumps({"objects": objects, "classifications": classifications}, indent=2)
 
     if state.get("json_output"):
-        Path("./ontology_output.json").write_text(json_ontology, encoding="utf-8")
-        rich.print("Stored mapping in [blue]`./ontology_output.json`")
+        output_file_path = Path("./ontology_output.json").resolve()
```

Comment:
```suggestion
        output_file_path = Path("./ontology_output.json").expanduser.absolute()
```

Reply:
.expanduser().absolute().resolve() is the best bet for full canonical path

Reply:
we can skip `expanduser` as the path is relative only and there is no `~` which expands the user and there is no need of `absolute` when you use `resolve` already.